cypress/integration/loans/loan-dates.spec.js

```diff
@@ -70,7 +71,10 @@ describe('loan dates', () => {
           })
           .then(() => {
             cy.login(userProperties.username, userProperties.password);
+            cy.visit(TopMenu.checkOutPath);
```

Comment:
add path to login() parameter to remove excess step with visit('/')

Reply:
Done